site/class-1-bar/maroeska/readme.md

```diff
@@ -0,0 +1,132 @@
+
+**catch** statement laat je de fout verwerken als programmeur.
+
+**errors** kunnen gecodeerde errors zijn die gemaakt zijn door de programmeur, zij kunnen het gevolg zijn van verkeerde invoer of andere onvoorziene zaken. 
+
+**return** statement stopt de uitvoering van een functie en geeft een waarde terug van die functie.
+
+**map** doorkruist de array van links naar rechts in, waarbij een callback (terugroep)functie op elk element wordt toegepast met parameters. Voor elke callback wordt de teruggegeven value een nieuw element in de nieuwe array. Nadat alle elementen zijn doorkruist map() geeft de nieuwe array alle vertaalde elementen terug.
+
+**d3.max** geeft de maximale waarde terug in het gegeven array met behulp van de natuurlijke volgorde. Als de array leeg is, keert het ongedefinieerd (undefined) terug. Een optionele accessor (toegangsfunctie) kan worden opgegeven, die gelijk is aan het oproepen van array.map (accessor) voordat je de maximale waarde berekent. 
+
+**axis** is de referentie schaal die overeenkomt met welke diagrammen zijn getekend. CanvasJS ondersteunt 4 assen types: primaire x-as, secundaire x-as, primaire y-as en secundaire y-as.
+axis heeft een path element met de class ‘domain’, gevolgd door getransformeerde g elementen van class ‘tick’. Elke tick heeft een line element, om de tick line te tekenen en een text element voor de tick label.
+
+**.call** wordt gebruikt om een functie aan te roepen met een eigen object als het eerste argument (parameter)
+
+**d3.axisBottom** construeert een nieuwe bodemgeoriënteerde asgenerator voor de gegeven schaal, met lege tick argumenten, een tick grootte van 6 en een padding van 3. In deze oriëntatie worden ticks onder de horizontale domain path getekend.
+
+**d3.axisLeft** construeert een nieuwe linker-georienteerde asgenerator voor de gegeven schaal, met lege tick argumenten, een tick grootte van 6 en een padding van 3. In deze oriëntatie worden ticks links van het verticale domain path getrokken.
+
+**dy**-attribuut geeft een verschuiving aan langs de y-as op de positie van een element of de inhoud ervan. Wat er exact verschuift hangt af van het element waarvoor dit attribuut is ingesteld.
+
+**text-anchor** wordt gebruikt om een tekenreeks van een bepaald punt in te stellen (begin-, midden- of eind-alignment).
+
+**.data** koppelt data van elk type aan DOM-elementen.
+
+**bandwith** geeft de breedte van elke band weer.
+
+Informatie bronnen:
+
+https://stackoverflow.com > diverse vragen van mensen gelezen<br>
+https://github.com/d3/d3-array > veel informatie gehaald over D3<br>
+https://developer.mozilla.org/nl/ > informatie gezocht over D3 en errors<br>
+https://developer.mozilla.org/nl/ > informatie gezocht over CSS en JS<br> 
+https://medium.com/@nick3499/d3-scaleband-rangeround-padding-ordinal-scale-with-range-bands-including-padding-f4af1e3c96ab > informatie gezocht over .Scaleband
+
+## License
+
+MIT © Maroeska Verkerk<br>
+Original files from Mike Bostock 
```

Comment:
Depending on where you found your example, Mike Bostock probably doesn’t allow you to use MIT!

[Check out the assignment again, and the note about licenses](https://github.com/cmda-fe3/course-17-18/blob/master/class-2.md).